subsys/bluetooth/audio/pacs.c

```diff
@@ -63,15 +64,57 @@ static uint16_t src_available_contexts = BT_AUDIO_CONTEXT_TYPE_PROHIBITED;
 static uint16_t src_supported_contexts = BT_AUDIO_CONTEXT_TYPE_PROHIBITED;
 #endif /* CONFIG_BT_PAC_SRC */
 
+enum {
+	FLAG_ACTIVE,
+#if defined(CONFIG_BT_PAC_SNK_NOTIFIABLE)
+	FLAG_SINK_PAC_CHANGED,
+#endif /* CONFIG_BT_PAC_SNK_NOTIFIABLE) */
+#if defined(CONFIG_BT_PAC_SNK_LOC_NOTIFIABLE)
+	FLAG_SINK_AUDIO_LOCATIONS_CHANGED,
+#endif /* CONFIG_BT_PAC_SNK_LOC_NOTIFIABLE */
+#if defined(CONFIG_BT_PAC_SRC_NOTIFIABLE)
+	FLAG_SOURCE_PAC_CHANGED,
+#endif /* CONFIG_BT_PAC_SRC_NOTIFIABLE */
+#if defined(CONFIG_BT_PAC_SRC_LOC_NOTIFIABLE)
+	FLAG_SOURCE_AUDIO_LOCATIONS_CHANGED,
+#endif
+	FLAG_AVAILABLE_AUDIO_CONTEXT_CHANGED,
+#if defined(CONFIG_BT_PACS_SUPPORTED_CONTEXT_NOTIFIABLE)
+	FLAG_SUPPORTED_AUDIO_CONTEXT_CHANGED,
+#endif /* CONFIG_BT_PACS_SUPPORTED_CONTEXT_NOTIFIABLE */
```

Comment:
Do we gain much from guarding these? 

Should `FLAG_AVAILABLE_AUDIO_CONTEXT_CHANGED` be guarded by `CONFIG_BT_PACS_SUPPORTED_CONTEXT_NOTIFIABLE`?

Reply:
> Do we gain much from guarding these?

I guess not very much. Since we already are guarding optional features like this, I prefer to have them, at least in my mind it gets a bit clearer.

> Should FLAG_AVAILABLE_AUDIO_CONTEXT_CHANGED be guarded by CONFIG_BT_PACS_SUPPORTED_CONTEXT_NOTIFIABLE?

No, because FLAG_AVAILABLE_AUDIO_CONTEXT_CHANGED  is not optional, and not dependent on supported contexts being notifiable or not.